packages/hop-node/src/watchers/HealthCheckWatcher.ts

```diff
@@ -675,12 +675,18 @@ export class HealthCheckWatcher {
 
   private async getChallengedTransferRoots (): Promise<ChallengedTransferRoot[]> {
     const result: any[] = []
+    const provider = getRpcProvider(Chain.Ethereum)!
+
+    // Challenges can only be made for the last 24 hours
+    const mainnetBlockTimeSec = 12
+    const challengeTimeWithBuffer = SecondsInDay * 1.25
+    const blocksInADay = challengeTimeWithBuffer / mainnetBlockTimeSec
+
     for (const token of this.tokens) {
       this.logger.debug(`done ${token} bridge for challenged roots`)
       const l1BridgeContract = contracts.get(token, Chain.Ethereum).l1Bridge
-      const provider = getRpcProvider(Chain.Ethereum)!
-      const startBlockNumber = 0
       const endBlockNumber = Number((await provider.getBlockNumber()).toString())
+      const startBlockNumber = endBlockNumber - blocksInADay
```

Comment:
Why not use `getBlockNumberFromDate` util to get block number from 24 hours ago?

Reply:
Updated in 3b3f8a3c0464023e8ac6bd8ec3a023029a169228